src/main.js

```diff
@@ -252,55 +252,46 @@ class MiniGraphCard extends LitElement {
   }
 
   renderStates() {
-    const { entity, value: tooltipValue } = this.tooltip;
-    let state;
     const [firstEntityConfig] = this.config.entities;
-    if (tooltipValue !== undefined) {
-      state = tooltipValue;
-    } else if (this.config.show.state === 'last') {
-      state = this.points[0][this.points[0].length - 1][V];
-    } else {
-      const [firstEntity] = this.entity;
-      if (firstEntityConfig.attribute) {
-        state = firstEntity.attributes[firstEntityConfig.attribute];
-      } else {
-        ({ state } = firstEntity);
-      }
-    }
-    const color = firstEntityConfig.state_adaptive_color ? `color: ${this.color};` : '';
     if (this.config.show.state)
       return html`
         <div class="states flex" loc=${this.config.align_state}>
-          <div class="state">
-            <span class="state__value ellipsis" style=${color}>
-              ${this.computeState(state)}
-            </span>
-            <span class="state__uom ellipsis" style=${color}>
-              ${this.computeUom(entity || 0)}
-            </span>
-            ${this.renderStateTime()}
-          </div>
-          <div class="states--secondary">${this.config.entities.map((ent, i) => this.renderState(ent, i))}</div>
+          ${this.renderState(firstEntityConfig, 0)}
+          <div class="states--secondary">${this.config.entities.map((ent, i) => i > 0 && this.renderState(ent, i) || '')}</div>
```

Comment:
I know this was already poorly named before but I would suggest

```suggestion
          <div class="states--secondary">${this.config.entities.map((entityConfig, i) => i > 0 && this.renderState(entityConfig, i) || '')}</div>
```